docs/guides/security/authentication/active_directory_authentication.md

```diff
@@ -183,3 +184,156 @@ To make this configuration change take effect, you must restart the
+
+## Interact with the AD using `adcli`
+
+`adcli` is a CLI to perform actions on an Active Directory domain.
+
+- If not yet installed, install the required package
+
+```sh
+[user@host ~]$ sudo dnf install adcli
+```
+
+- Test if you have ever joigned an Active Directory domain:
+
+```sh
+[user@host ~]$ sudo adcli testjoin
+Successfully validated join to domain ad.company.local
+```
+
+- Get more advanced informations about the domain:
+
+```sh
+[user@host ~]$ adcli info ad.company.local
+[domain]
+domain-name = ad.company.local
+domain-short = AD
+domain-forest = ad.company.local
+domain-controller = dc1.ad.company.local
+domain-controller-site = site1
+domain-controller-flags = gc ldap ds kdc timeserv closest writable full-secret ads-web
+domain-controller-usable = yes
+domain-controllers = dc1.ad.company.local dc2.ad.company.local
+[computer]
+computer-site = site1
+```
+
+- More than a consulting tool, you can use adcli to interact with your domain: manage users or groups, change password, etc. 
+
+Example: use `adcli` to get information about a computer:
+
+!!! Note
+
+    This time we will provide an admin username thanks to the `-U` option
+
+```sh
+[user@host ~]$ adcli show-computer pctest -U admin_username
+Password for admin_username@AD: 
+sAMAccountName:
+ pctest$
+userPrincipalName:
+ - not set -
+msDS-KeyVersionNumber:
+ 9
+msDS-supportedEncryptionTypes:
+ 24
+dNSHostName:
+ pctest.ad.company.local
+servicePrincipalName:
+ RestrictedKrbHost/pctest.ad.company.local
+ RestrictedKrbHost/pctest
+ host/pctest.ad.company.local
+ host/pctest
+operatingSystem:
+ Rocky
+operatingSystemVersion:
+ 8
+operatingSystemServicePack:
+ - not set -
+pwdLastSet:
+ 133189248188488832
+userAccountControl:
+ 69632
+description:
+ - not set -
+```
+
+Example: use `adcli` to change user's password:
+
+```sh
+[user@host ~]$ adcli passwd-user user_test -U admin_username
+Password for admin_username@AD: 
+Password for user_test: 
+[user@host ~]$ 
+```
+
+## Troubleshooting
+
+Sometimes, the network service will start after sssd, that cause trouble with authentication.
+
+No AD users will be able to connect until you restarted the service.
+
+In that case, you will have to override the systemd's service file to manage this problem.
+
+Copy this content into `/etc/systemd/system/sssd.service`:
+
+```
+[Unit]
+Description=System Security Services Daemon
+# SSSD must be running before we permit user sessions
+Before=systemd-user-sessions.service nss-user-lookup.target
+Wants=nss-user-lookup.target
+After=network-online.target
+
+
+[Service]
+Environment=DEBUG_LOGGER=--logger=files
+EnvironmentFile=-/etc/sysconfig/sssd
+ExecStart=/usr/sbin/sssd -i ${DEBUG_LOGGER}
+Type=notify
+NotifyAccess=main
+PIDFile=/var/run/sssd.pid
+
+[Install]
+WantedBy=multi-user.target
+```
+
+The next reboot, the service will start after it's requirements, and everything will go well.
```

Comment:
I'm not sure exactly what this phrase means or if there is a better expression than "after it's requirements" but either way, the word is "its" without an apostrophe.